sw/device/tests/BUILD

```diff
@@ -1115,21 +1108,15 @@ opentitan_test(
     name = "edn_kat",
     srcs = ["edn_kat.c"],
     # Remove this line when fixed.
-    broken = cw310_params(tags = ["broken"]),
     exec_env = dicts.add(
         EARLGREY_TEST_ENVS,
         {
             "//hw/top_earlgrey:silicon_creator": None,
             "//hw/top_earlgrey:silicon_owner_sival_rom_ext": "silicon_owner",
             "//hw/top_earlgrey:silicon_owner_prodc_rom_ext": "silicon_owner",
             "//hw/top_earlgrey:silicon_owner_proda_rom_ext": "silicon_owner",
```

Comment:
I think these need replacing with `EARLGREY_SILICON_OWNER_ROM_EXT_ENVS` now that the `silicon_owner = ...` params are deleted